nixos/tests/matrix-synapse.nix

```diff
@@ -8,23 +36,31 @@ import ./make-test.nix ({ pkgs, ... } : {
   nodes = {
     # Since 0.33.0, matrix-synapse doesn't allow underscores in server names
     serverpostgres = args: {
-      services.matrix-synapse.enable = true;
-      services.matrix-synapse.database_type = "psycopg2";
+      services.matrix-synapse = {
+        enable = true;
+        database_type = "psycopg2";
+        tls_certificate_path = "${cert}";
```

Comment:
```suggestion
        tls_certificate_path = cert";
```

Reply:
I got an error with this as `cert` is a path and `tls_certificate_path` wants a string

Reply:
Makes sense :)